hack/verify-labels.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TESTINFRA_ROOT=$(git rev-parse --show-toplevel)
+
+TMP_OUTPUT=$(mktemp)
```

Comment:
`$(mktemp -d)`

Reply:
Aside: I think both of these nits also exist in verify-config.

Reply:
this is intentionally a file, I'll rename to make that more clear